everyvoice/tests/regression/prep-datasets.sh

```diff
@@ -70,6 +79,7 @@ uqeqesho
 iimpumlo
 ==EOF==
 echo isiXhosa > "$dir"/test2.txt
+done
```

Comment:
Same here, the literal text going into test.txt, and the `==EOF==` have to sit against the left margin, but not the bash code.